content/docs/getting-started/connect-database.md

```diff
@@ -43,6 +43,20 @@ nuosql "demo@${DB_URL}:8443" --user dba --password changeIt --connection-propert
 
 [NuoDB client](https://github.com/nuodb/nuodb-client/releases) package v20230228 or later is required to connect to DBaaS database.
 
+{{< /tab >}}
+
+{{< tab "terraform" >}}
+
+```terraform

```

Comment:
The example might need some explanation. It defines TF module outputs from the created database.
I suggest adding all the outputs needed for an app to connect. For example:

```terraform
output "db_url" {
  value = "${nuodbaas_database.db.status.sql_endpoint}:443"
}

output "db_name" {
  value = nuodbaas_database.db.name
}

output "dba_user" {
  value = "dba"
}

output "dba_password" {
  value = random_password.password.result
  sensitive = true
}
output "ca_cert" {
  value = nuodbaas_database.db.status.ca_pem
}


```